plenigo_plugin/wooCommerce/WC_Gateway_Plenigo.php

```diff
@@ -34,6 +33,8 @@ class WC_Gateway_Plenigo extends \WC_Payment_Gateway {
      * Holds the values to be used in the fields callbacks
      */
     private $options = null;
+    // Available Product Types
+    private $prodTypeList = array('EBOOK', 'DIGITALNEWSPAPER', 'DOWNLOAD', 'VIDEO', 'MUSIC');
```

Comment:
I understand that a import of a third party file is not good, but when it is a file of our project it should not be a problem, or i am wrong?